Update the WDL model

mhouppin opened the pull request:
Make the WDL model use the material value instead of the game ply.

Used 10k selfplay games at 80knpm, using 4moves_noob as the opening book. See https://github.com/official-stockfish/WDL_model for precise information on how the model works.

Bench: 4,139,804

![scoreWDL](https://github.com/mhouppin/stash-bot/assets/45004383/a9f91045-6abd-477a-b909-3d27b542bca3)

EarlyEdition commented:
Wonderful! @mhouppin Congrats on a great idea :)

I think we can merge now.

mhouppin commented:
Dropped the data at 40+0.4, and switched to 10k games at 8+0.08 using Pohl.epd as the opening book.

![scoreWDL](https://github.com/mhouppin/stash-bot/assets/45004383/4b91bb75-a375-4021-b919-a6cbdad24c32)
